docs/en/query_language/functions/date_time_functions.md

```diff
@@ -311,22 +311,58 @@ SELECT
 └──────────────────────────┴───────────────────────────────┘
 ```
 
-## dateDiff('unit', t1, t2, \[timezone\])
-
-Return the difference between two times expressed in 'unit' e.g. `'hours'`. 't1' and 't2' can be Date or DateTime, If 'timezone' is specified, it applied to both arguments. If not, timezones from datatypes 't1' and 't2' are used. If that timezones are not the same, the result is unspecified.
-
-Supported unit values:
-
-| unit   |
-| ------ |
-|second  |
-|minute  |
-|hour    |
-|day     |
-|week    |
-|month   |
-|quarter |
-|year    |
+## dateDiff {#datediff}
+
+Returns the difference between two time values.
+
+**Syntax**
+
+```sql
+dateDiff('unit', startdate, enddate, [timezone])
+```
+
+**Parameters**
+
+- `unit` — Time unit, in which the returned value is expressed. [String](../syntax.md#syntax-string-literal). 
+
+    Supported values:
+
+    | unit   |
+    | ------ |
+    |second  |
+    |minute  |
+    |hour    |
+    |day     |
+    |week    |
+    |month   |
```

Comment:
Needs detailed explanation on how exactly does the months and beyond work.
For example, difference between December 31 and January 1 is one month or zero?